test/test_torch.py

```diff
@@ -3698,7 +3698,7 @@ def test_randint_inference(self):
         for args in [(3,), (1, 3)]:  # (low,) and (low, high)
             self.assertIs(torch.int64, torch.randint(*args, size=size).dtype)
             self.assertIs(torch.int64, torch.randint(*args, size=size, layout=torch.strided).dtype)
-            self.assertIs(torch.int64, torch.randint(*args, size=size, generator=torch.default_generator).dtype)
+            self.assertIs(torch.int64, torch.randint(*args, size=size, generator=torch.random.default_generator).dtype)
```

Comment:
Is this a BC-breaking change? Or is `torch.default_generator` still valid?

Reply:
This would be a BC-breaking change :\. Removed it from: https://github.com/pytorch/pytorch/blob/master/torch/csrc/Module.cpp#L675. Bringing this back through torch/__init__.py